ui/src/app/components/dataflow-form/dataflow-form.component.html

```diff
@@ -1,38 +1,109 @@
+          </mat-form-field>
+          <br>
+          <mat-form-field class="full-width" appearance="outline" matTooltip="The machine type to use for the job, eg: n1-standard-2. Use default machine type if not specified."
+          [matTooltipPosition]="'right'">
+            <mat-label>Machine Type</mat-label>
+            <input matInput placeholder="Machine Type" type="text" formControlName="machineType"/>
+          </mat-form-field>
+          <h5>Find the list of all machine types <a href="https://cloud.google.com/compute/docs/machine-resource" target="_blank">here</a>.</h5>
+        </div>
+      </mat-expansion-panel>
+      <br>
+      <mat-form-field class="full-width" appearance="outline" matTooltip="Set the service account to run the dataflow job(s) as"
+      [matTooltipPosition]="'right'">
+        <mat-label>Service Account Email</mat-label>
+        <input matInput placeholder="Service Account Email" type="text" formControlName="serviceAccountEmail"/>
       </mat-form-field>
       <br>
-      <mat-form-field class="full-width" appearance="outline">
-        <mat-label>VPC Network</mat-label>
-        <input matInput placeholder="VPC Network Name" type="text" formControlName="network"/>
+      <mat-form-field class="full-width" appearance="outline" matTooltip="Additional user labels to be specified for the job. Enter a json of &quot;key&quot;: &quot;value&quot; pairs. Example: {&quot;name&quot;: &quot;wrench&quot;, &quot;mass&quot;: &quot;1kg&quot;, &quot;count&quot;: &quot;3&quot; }."
+      [matTooltipPosition]="'right'">
+        <mat-label>Additional User Labels</mat-label>
+        <input matInput placeholder="Additional User Labels" type="text" formControlName="additionalUserLabels"/>
       </mat-form-field>
       <br>
-      <mat-form-field class="full-width" appearance="outline">
-        <mat-label>VPC Subnetwork</mat-label>
-        <input matInput placeholder="VPC Subnetwork Name" type="text" formControlName="subnetwork"/>
+      <mat-form-field class="full-width" appearance="outline" matTooltip="Name for the Cloud KMS key for the job. Key format is: projects/<project>/locations/<location>/keyRings /<keyring>/cryptoKeys/<key>. Omit this field to use Google Managed Encryption Keys."
+      [matTooltipPosition]="'right'">
+        <mat-label>KMS Key Name</mat-label>
+        <input matInput placeholder="KMS Key Name" type="text" formControlName="kmsKeyName"/>
       </mat-form-field>
+    </form>
+      <hr>
+    <form [formGroup]="presetFlagsForm" class="dataflow-form">
+      <h2>
+        Preset Flags 
+        <span><button mat-button class="configure" (click)="enablePresetFlags()" [disabled]="!disablePresetFlags">
+          <mat-icon>edit</mat-icon>
+          EDIT
+      </button></span>
+      </h2> 
+      <h5>These flags are set by SMT by default and <b>SHOULD NOT BE</b> modified unless running Dataflow in a non-standard configuration.</h5>
```

Comment:
These should look disabled, not ever clear from the screenshot if they are disabled or not. Also, when we hover or them, does the cursor change to disabled?

Reply:
This is what the presets look like:
<img width="524" alt="Screenshot 2023-10-25 at 1 53 50 PM" src="https://github.com/GoogleCloudPlatform/spanner-migration-tool/assets/19630018/26f5d732-c45c-46de-be97-614b5ad24bbf">

The font color for disabled fields is slightly different (not too contrasting I agree). The user anyways cannot edit these fields even if they try till they click on the edit button (since they are disabled by default). So there is no way for the user to accidentally modify these fields.